pkg/sbproxy/reconcile/reconcile_brokers.go

```diff
@@ -81,102 +113,100 @@ func (r ReconcileBrokersTask) run() {
 
 // reconcileBrokers attempts to reconcile the current brokers state in the platform (existingBrokers)
 // to match the desired broker state coming from the Service Manager (payloadBrokers).
-func (r ReconcileBrokersTask) reconcileBrokers(existingBrokers []serviceBrokerReg, payloadBrokers []serviceBrokerReg) {
+func (r ReconcilationTask) reconcileBrokers(existingBrokers []platform.ServiceBroker, payloadBrokers []platform.ServiceBroker) {
```

Comment:
They slightly differ (e.g. platform brokers have a name that we use as well as part of the delete request. I could embed sm broker into platform broker but this will get us cycle deplendencies. I will leave it like this for now